manage.py

```diff
@@ -188,3 +189,20 @@ def translation_download(language, state):
     state = state or default_state
     click.echo('Downloading {state} translations from Smartling'.format(state=state))
     smartling_download(state=state, language=language)
+
+
+@click.option('--config_key', '-c', help='Return only a single config value')
+@app.cli.command()
+def config(config_key):
+    """List current flask configuration values in JSON"""
+
+    if config_key:
+        # Remap None values to an empty string
+        print(app.config.get(config_key, '') or '')
+        return
+
+    print(json.dumps(
+        # Skip un-serializable values
+        {k: v for k, v in app.config.items() if isinstance(v, basestring)},
```

Comment:
Hmm, what types were you running into problems with?  This is silently omitting them, should we print out a string representation, or are you confident they aren't needed?

Reply:
There are some config values that are objects (`SESSION_REDIS`); I don't think we're missing any important ones